mozilla_aws_cli/login.py

```diff
@@ -381,17 +357,42 @@ def exchange_token_for_credentials(self):
                 logger.debug('Unable to assume role {}'.format(self.role_arn))
                 self.cache = False
                 self.get_role_map()
+                if self.batch:
+                    self.exit(
+                        "Unable to assume role. Shutting down due to batch "
+                        "mode being enabled")
+                    return 'error'
+                self.state = "role_picker"
```

Comment:
I think we should probably have a space above this line, since we're now moving to a new state.

Reply:
Done in 4c19b5d43a469e4c2fccef8538be1c2f00c7cb31